public/sw.js

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable no-unused-vars */
+const VERSION = 1
```

Comment:
Ok, so it looks weird but basically the way a service worker operates is that it checks for the byte sum of the file and if it has changed, determines that it is a new service worker to be installed. Having the version defined (though not used) enables us to easily change the byte sum of the file and therefore trigger an update.

Lots of people use the service worker version in the cache name, but because we also want to access the cache from the app (which is basically a separate code base since this is in `/public`) I didn't want to have a cache mismatch because we forgot to update the version number in two places.

Happy to change, not sure what the best strategy is here though.